lib/matplotlib/axis.py

```diff
@@ -1825,8 +1825,11 @@ def set_ticklabels(self, ticklabels, *, minor=False, **kwargs):
             For each tick, includes ``tick.label1`` if it is visible, then
             ``tick.label2`` if it is visible, in that order.
         """
-        ticklabels = [t.get_text() if hasattr(t, 'get_text') else t
-                      for t in ticklabels]
+        try:
+            ticklabels = [t.get_text() if hasattr(t, 'get_text') else t
+                         for t in ticklabels]
```

Comment:
So, line up the "f" in `for` with the "t" in `t.get_text()`.

Reply:
fixed

Reply:
> The other errors are happening for reasons outside our control and is getting worked on separately.


Oh :|

Reply:
@WeatherGod Are all issues on my end fixed?